account_tax_rounding_method/models/account_move.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from collections import defaultdict
+from contextlib import contextmanager
+
+from odoo import api, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        # Group vals by partner_id
+        grouped_vals = defaultdict(list)
+        for vals in vals_list:
+            partner_id = vals.get("partner_id")
+            grouped_vals[partner_id].append(vals)
+        created_recs = self.browse()
+        for partner_id, partner_vals in grouped_vals.items():
+            context = self.env.context
+            if partner_id:
+                partner = self.env["res.partner"].browse(partner_id)
+                if partner.tax_rounding_method:
+                    context = dict(
+                        context, tax_rounding_method=partner.tax_rounding_method
+                    )
```

Comment:
Please check the validity of this suggestion.
```suggestion
            partner = self.env["res.partner"].browse(partner_id or False)
            tax_rounding_method = self.env["account.tax"]._get_tax_rounding_method
            if tax_rounding_method:
                context = dict(context, tax_rounding_method=tax_rounding_method)
```